app/Http/Controllers/PlannerController.php

```diff
@@ -11,9 +12,12 @@ class PlannerController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Planner $planner)
     {
-        //
+        $planner = Planner::all();
```

Comment:
Later on you should change this so it only shows the users planners and not everyone's planners.